### PR TITLE
Fixing errors

### PR DESCRIPTION
**Pull request by allopez7**

### CODE REVIEW

**Comment by Wolfy64** [comment]
Do not forget to `npm run autofix` before to push 🚀

File: src/api/admin.ts [hunk 1]
```diff
@@ -12,11 +12,11 @@ const exec = util.promisify(cp.exec)
 app.post('/sslCerts', async (req, res) => {
   try {
     const serviceKeys = (getData('serviceKeys') || []).filter(
-      d => d.service === req.body.service
+      (d: any) => d.service === req.body.service
```
Comment (songz) [inline src/api/admin.ts]: should be `(d: ServiceKey)` no?

Also, could you change `getData` to `getProviders` or something? I think there was a helper function that returns the correct type.

File: src/providers/goDaddy.ts [hunk 2]
```diff
@@ -5,8 +5,8 @@ import { ServiceKey } from '../api/types/admin'
 export const getDomains = () => {
   const serviceKeys: any = getData('serviceKeys')
```
Comment (songz) [inline src/providers/goDaddy.ts]: use helper function that returns the correct type

**Comment by songz** [comment]
This is resolved by this PR: https://github.com/garageScript/myProxy/pull/56/files

**Pull request closed without merging**